pkg/sfu/streamallocator/streamallocator.go

```diff
@@ -1297,7 +1297,7 @@ func (s *StreamAllocator) initProbe(probeGoalDeltaBps int64) {
 	s.channelObserver = s.newChannelObserverProbe()
 	s.channelObserver.SeedEstimate(s.lastReceivedEstimate)
 
-	s.params.Logger.Infow(
+	s.params.Logger.Debugw(
```

Comment:
Need to make analytics events for these.

Reply:
yeah I left the other one alone because they contained information about the change. but seems like notifying that we are starting to probe is only useful in limited situations. would be great to have `congestion_started` and `congestion_ended` events